packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -131,7 +131,11 @@ ${buildWarnEnvConflicts(edge, runtimeDir, runtimeName)}
 ${buildDebugInitialization(edge)}
 const PrismaClient = getPrismaClient(config)
 exports.PrismaClient = PrismaClient
-Object.assign(exports, Prisma)
+Object.assign(exports, Prisma)${
+      this.options.generator?.previewFeatures.includes('denoDeploy')
```

Comment:
I think this could be a problem once the preview fades away. I think we mentioned we could work around that by generating special deno entrypoint to be imported like `.prisma/client/deno/edge.ts`

Reply:
(Solution in comments below)

Reply:
https://github.com/prisma/prisma/pull/14386/files#r960846120